README.md

```diff
@@ -1,34 +1,118 @@
-# BotFlow - Plataforma de Criação e Edição de conteúdos de chatbot Rasa.
+# BotFlow-Platform for creating and editing contents of Rasa chatbot.
 
-## Sobre o Projeto
-BotFlow é uma plataforma desenvolvida pelo LAPPIS - Laboratório Avançado de Produção, Pesquisa e Inovação em Software (FGA/UnB),  em parceria com o Ministério da Cultura para facilitar a criação e edição de conteúdos inseridos em chatbots desenvolvidos por meio da implementação do framework Rasa como a Tais, um chatbot desenvolvido para o projeto da Lei Rouanet.
+# # About the project
+BotFlow is a platform developed by LAPPIS-Advanced laboratory of production, research and innovation in Software (FGA/UnB), in partnership with the Ministry of Culture to facilitate the creation and editing of content inserted in chatbots developed through the Implementation of the RASA framework such as Tais, a chatbot developed for the project of the Rouanet law.
 
-<!-- Links uteis: -->
-* **O que é o Lappis? ✏️** [Conheça o Lappis](https://lappis.rocks)
-* **O que é a Tais? 🤔** [Conheça a Tais](#o-que-é-a-tais)
+<!-- Useful Links: -->
+* **What is Lappis? ✏️** [Meet Lappis](https://lappis.rocks)
+* **What is Tais? 🤔** [Meet Tais](https://github.com/lappis-unb/tais)
+* **What is Rasa?** [Meet Rasa](https://blog.rasa.com/)
 
 
-## Funcionalidades BotFlow
-* Criação e edição de Perguntas
-* Criação e edição de Respostas
-* Criação e edição de fluxos de conversa
-* Teste de interação referente ao conteúdo criado ou editado
+# # BotFlow Features
```

Comment:
Melhor apresentar a terminologia do Rasa primeiro, depois a nossa. Ex: "Create and edit *intents*, called *questions*"